.github/workflows/build.yml

```diff
@@ -46,7 +46,7 @@ jobs:
     # and co-authored by the user that made the last commit.
     # https://github.com/marketplace/actions/git-auto-commit
     - name: Commit the changes to the binary files
-      if: false  # node-gyp builds are not reproducible, hence each build would create a "new" commit -> disabling for now (comment out this line if needed)
```

Comment:
Sorry for the confusing commented out line - it should say `if: true`. See 6bdce42 in #21.